File: ex1_calculo_media/main.py
# Entrada de dados e conversão explicita para float
nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))
nota3 = float(input("Digite a terceira nota: "))

# Cálculo da média com parênteses para garantir a ordem correta
# (soma primeiro, depois divisão)
media = (nota1 + nota2 + nota3) / 3

# Expressão relacional que gera um valor booleano
aprovado = media >= 7

# Saída
print(f"Média: {media:.2f}")
if aprovado:
    print("Aprovado!")
else:
    print("Reprovado!")

File: ex1_calculo_media/main.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double nota1, nota2, nota3;

    cout << "Digite a primeira nota: ";
    cin >> nota1;
    cout << "Digite a segunda nota: ";
    cin >> nota2;
    cout << "Digite a terceira nota: ";
    cin >> nota3;

    // Parênteses garantem soma antes da divisão
    double media = (nota1 + nota2 + nota3) / 3.0;

    bool aprovado = media >= 7.0;

    cout << fixed << setprecision(2);
    cout << "Média: " << media << endl;
    if (aprovado) {
        cout << "Aprovado!" << endl;
    } else {
        cout << "Reprovado!" << endl;
    }

    return 0;
}